app/views/forms/payment_link/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.payment_link_form'), @payment_link_form.errors.any?)) %>
+<% content_for :back_link, govuk_back_link_to(form_path, t("back_link.form_create")) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <%= form_with(model: @payment_link_form, url: payment_link_path) do |f| %>
+      <% if @payment_link_form&.errors.any? %>
+        <%= f.govuk_error_summary %>
+      <% end %>
+
+      <span class="govuk-caption-l"><%= @payment_link_form.form.name %></span>
+      <h1 class="govuk-heading-l">
+        <%= t('payment_link_form.heading') %>
+      </h1>
+
+      <%= t('payment_link_form.body_html') %>
+
+      <h2 class="govuk-heading-m">
+        <%= t('payment_link_form.how_this_will_work.heading') %>
+      </h2>
```

Comment:
What's the reason for not putting the `h2` headers in the translation file?

Reply:
Ooh not sure, @DavidBiddle any thoughts?

Reply:
Maybe just personal preference but I generally try to keep as much of the HTML structure as is reasonable in the view file, and have the translation file contains smaller blocks of content. I guess my thinking is mainly that this should help make the locale files easier for non-devs to update.

Reply:
Interesting, that makes sense. Thanks for explaining!